app/manifest.webapp

```diff
@@ -7,7 +7,7 @@
     "48": "/img/omrs-button.png"
   },
   "developer": {
-    "name": "andela-odaniel, wanjikum, andela-jomadoye, andela-pupendo, kingisaac95",
```

Comment:
Is there a reason you removed the names of these people? They also worked on the project

Reply:
I realized that these are the names that appear on the module-refapp when we upload a zip file. Considering that it is just the two of us currently working on the module, it made sense to update the names, such that if someone was to access the refapp and wish to contact those working on the project, they would know who made the most recent uploads/who is currently working on the project. 
I realized the other teams also just have the current members working on a project listed.  It makes more sense, rather than having a long list of people most of whom have left the project.
What are your thoughts on this?

Reply:
I suggest `Andela Team`

Reply:
@mkiterian if you look at the format followed by others, you will notice we use the names that  people use on `talk`, which then means that someone testing out the module on refapp should know who to reach out to on talk, should they see an issue in it. Therefore, `Andela Team` would not suit that context.

@dkayiwa What are your thoughts on this?

Reply:
Since the andela devs working on this keep changing, for maintenance purposes, Andela Team looks easier.

Reply:
Okay,  thank you.

Reply:
Resolved.